lib/rets/metadata/lookup_type.rb

```diff
@@ -4,8 +4,8 @@ class LookupType
       attr_reader :long_value, :value
 
       def initialize(lookup_type_fragment)
-        @value       = lookup_type_fragment["Value"]
-        @long_value  = lookup_type_fragment["LongValue"]
+        @value       = lookup_type_fragment["Value"].rstrip
+        @long_value  = lookup_type_fragment["LongValue"].rstrip
```

Comment:
every other `strip` in this codebase is just a standard `strip`, not `rstrip`. What's the reason to switch here?